__tests__/app.test.js

```diff
@@ -365,4 +365,27 @@ describe('APP', () => {
             })
         });
     });
+    describe('DELETE /api/comments/:comment_id', () => {
+        test('should respond with a 204 status and delete the comment with requested id', () => {
```

Comment:
this is great and all we need to do - the fact that nothing has come back from the data base (including no error) means the delete was successful - if you really want to be assured that the comment was deleted you could do a query here and expect there to be no comment in the database with that id but this is a bit overkill and unnecessary